tests/unit_tests/encoders/text/test_vocab.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+
+from lightwood.encoders.text.vocab import VocabularyEncoder
+
+
+class TestVocabularyEncoder(unittest.TestCase):
+    def test_encode_decode(self):
+        self.maxDiff = None
```

Comment:
Currently useless

Reply:
useful since the text diffs can be large, and `maxDiff` set to the magic value allows us to see them.